ingester/ingester.go

```diff
@@ -515,7 +515,7 @@ func (i *Ingester) flushSeries(u *userState, fp model.Fingerprint, series *memor
 	}
 
 	firstTime := series.chunkDescs[0].FirstTime()
-	flush := immediate || model.Now().Sub(firstTime) > i.cfg.MaxChunkAge
+	flush := immediate || len(series.chunkDescs) > 1 || model.Now().Sub(firstTime) > i.cfg.MaxChunkAge
```

Comment:
> Why strictly greater than one?

We need to flush something in the series if:
- we are existing (`immedidate`)
- a chunk has be filled by the incoming data and a new one created (`len > 1`)
- the series (and by implication, the one and only chunk) is older than the threshold